io.asgardio.dotnet.oidc.sdk.sample/App.config

```diff
@@ -0,0 +1,85 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Copyright (c) 2019, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+    
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    
+    http://www.apache.org/licenses/LICENSE-2.0
+    
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License. -->
```

Comment:
License header is wrong

Reply:
Formatting is resolved in https://github.com/wso2-extensions/identity-sdks-dotnet/pull/6/commits/ad95c5751c2becdbce174d985942960300fed9bc.
However, the didn't change the year as this was a file created when migrating the Dotnet SDK sample from https://github.com/wso2-extensions/identity-samples-dotnet